admin/pages/tools.php

```diff
@@ -53,11 +51,15 @@
 	asort( $tools );
 
 	echo '<ul class="ul-disc">';
+
+	$admin_url = admin_url( 'admin.php?page=wpseo_tools' );
+
 	foreach ( $tools as $slug => $tool ) {
-		$href = ( ! empty( $tool['href'] ) ) ? esc_attr( $tool['href'] ) : '&tool=' . $slug;
+		$href = ( ! empty( $tool['href'] ) ) ? $admin_url . $tool['href'] : add_query_arg( array( 'tool' => $slug ) , $admin_url );
```

Comment:
The total of `$href` will be escaped on line [62](https://github.com/Yoast/wordpress-seo/pull/3617/files#diff-ff83250bc5314cfa4e842c300c16b157R62)